src/panels/lovelace/components/hui-image.ts

```diff
@@ -40,6 +40,8 @@ export class HuiImage extends LitElement {
 
   @property() public aspectRatio?: string;
 
+  @property() public refreshRate?: string;
```

Comment:
This should be a number, not a string

Reply:
I originally coded it with "number", however the value was never saved. Maybe I missed somewhere I needed to change.